### ISSUE TITLE
Update exporters to include classification tags

### ISSUE THREAD

**Issue by crubier** [labelflow/labelflow#485]
Continuation of https://github.com/labelflow/labelflow/issues/446 regarding exports.

Separated in new issue to avoid bloat.

- [ ] Update COCO export resolver to export classification tags
  - [ ] For each tag, have a bounding box on the complete image with the corresponding class
- [ ] Update YOLO export resolver to export classification tags
  - [ ] For each tag, have a bounding box on the complete image with the corresponding class

Blocked by #447

**Comment by nicOwlas** [comment]
Closing as tags are already exported as bounding boxes covering the entire image for both export format.
